go/arrow/csv/csv.go

```diff
@@ -237,6 +238,11 @@ func (r *Reader) validate(recs []string) {
 
 func (r *Reader) read(recs []string) {
 	for i, str := range recs {
+		if str == "" {
+			r.bld.Field(i).AppendNull()
```

Comment:
actually, there's an issue here: how do we make the distinction b/w a `null` value for a `string` column and a valid `""` value for a `string` column?

Reply:
thanks. I fixed this.